### PR TITLE
feat(apis_entities): add `changed since` filter to entity filterset

### PR DESCRIPTION
**Pull request by b1rger**
Closes: #335

### CODE REVIEW

File: apis_core/apis_entities/filtersets.py [hunk 1]
```diff
@@ -53,6 +56,12 @@ def related_entity(queryset, name, value):
     return queryset.filter(pk__in=t)
 
 
+def changed_since(queryset, name, value):
+    history = get_history_manager_for_model(queryset.model)
+    ids = history.filter(history_date__gt=value).values_list("pk", flat=True)
+    return queryset.filter(pk__in=ids)
```
Comment (sennierer) [inline apis_core/apis_entities/filtersets.py:62]: This needs to be changed to:

```suggestion
    ids = history.filter(history_date__gt=value).values_list("id", flat=True)
```

`pk` is the primary key of the version, `id` is the primary key of the db instance

**Review by sennierer — CHANGES_REQUESTED** [review]

File: apis_core/apis_entities/filtersets.py (hunk 1 above)
Comment (sennierer) [reply apis_core/apis_entities/filtersets.py:62]: sorry my IDE again changed the lines.
line 61 needs to be changed to:

```python
ids = history.filter(history_date__gt=value).values_list("id", flat=True)
```

Comment (b1rger) [reply apis_core/apis_entities/filtersets.py:118]: fixed

**Review by sennierer — APPROVED** [review]
working as intended

**Pull request merged**